Marlin/language_en.h

```diff
@@ -120,6 +120,15 @@
 #ifndef MSG_MOVE_E
 #define MSG_MOVE_E                          "Extruder"
 #endif
+#ifndef MSG_MOVE_E1
+#define MSG_MOVE_E1                         "Extruder1"
+#endif
+#ifndef MSG_MOVE_E2
+#define MSG_MOVE_E2                         "Extruder2"
+#endif
+#ifndef MSG_MOVE_E3
+#define MSG_MOVE_E3                         "Extruder3"
+#endif
```

Comment:
I think we should define `MSG_MOVE_E` differently with this update, replacing it with `MSG_MOVE_E0`:
```
#if EXTRUDERS < 2
  #define MSG_MOVE_E0                       "E"
#else 
  #define MSG_MOVE_E0                       "E0"
#endif
```